portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +50,48 @@ function showPicture(){
   imageContainer.appendChild(imgElement);    
 }
 
-function sayHello(){
-  fetch('/data').then(response => response.text()).then((statement) => {
-    document.getElementById('server-container').innerText = statement;
+function getDatastoreComments(){
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const serverContainer = document.getElementById('server-container');
+    comments.forEach((comment) => {
+      serverContainer.appendChild(createCommentBox(comment));
+    })
   });
 }
+
+/**
+ * Creates a small box which shows a user's name and comment.
+ * Also includes a button to delete the box and the information
+ * from the datastore. 
+ */
+function createCommentBox(comment) {
+  const boxElement = document.createElement('div');
+  boxElement.className = 'boxes';
+
+  const nameElement = document.createElement('strong');
+  nameElement.innerText = comment.userName;
+  nameElement.style.textDecoration = "underline";
+  nameElement.style.textAlign = 'center';
```

Comment:
Can you move the styling from this function into CSS?

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns